aidants_connect_web/views/service.py

```diff
@@ -53,22 +55,56 @@ def habilitation(request):
     return render(request, "public_website/habilitation.html")
 
 
+def mandats_chart(request):
+    demarches_count = []
+
+    autorisation_use = Journal.objects.excluding_staff().filter(
+        action="use_autorisation"
+    )
+
+    for demarche in settings.DEMARCHES.keys():
+        demarches_count.append(
+            {
+                "title": demarche,
+                "icon": settings.DEMARCHES[demarche]["icon"],
+                "value": autorisation_use.filter(demarche=demarche).count(),
+            }
+        )
```

Comment:
On peut même se passer de pré-instancier une liste vide, car Python est beau et doux : 
```suggestion
    demarches_count = [{
        "title": demarche,
        "icon": settings.DEMARCHES[demarche]["icon"],
        "value": autorisation_use.filter(demarche=demarche).count(),
    } for demarche in settings.DEMARCHES.keys()]
```